jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -112,7 +113,8 @@
             [ "Cloud:OpenStack:Master", "Cloud:OpenStack:Pike:Staging" ].contains(project) &&
             [
               "openstack-horizon-plugin-murano-ui",
-              "openstack-gnocchi"
+              "openstack-gnocchi",
+              "python-ironic-python-agent"
```

Comment:
this actually will exclude the tracking in :Master and :Pike:Staging, you can see that here:

https://ci.opensuse.org/view/OpenStack/job/openstack-trackupstream/2318/matrix-reloaded/

Reply:
@aplanas fixed. Could you have another look please?